core/templates/dev/head/services/local-storage.service.ts

```diff
@@ -132,7 +132,7 @@ export class LocalStorageService {
    *   translation if it exists, else null.
    */
   getLastSelectedTranslationLanguageCode(): string | null {
-    if (this.storage) {
+    if (this.isStorageAvailable()) {
```

Comment:
Just a note: Here I had to change the condition so I could spy on `isStorageAvailable` and then handle the else condition. I've tried to spy `storage` property but none of my tries worked.